any_test.go

```diff
@@ -0,0 +1,22 @@
+package cosmos_proto_test
+
+import (
+	"testing"
+
+	"google.golang.org/protobuf/proto"
+	_ "google.golang.org/protobuf/types/known/anypb"
```

Comment:
If I add this this, then I get a registry conflict error, as expected:

```
panic: proto: file "google/protobuf/any.proto" has a name conflict over google.protobuf.Any
	previously from: "github.com/cosmos/cosmos-proto"
	currently from:  "google.golang.org/protobuf/types/known/anypb"
See https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict
```

Reply:
The question is whether or not this is inevitable because of other imports bringing in the official anypb. My sense is that it's a bit risky. Also this approach will require users that use the official go proto plugin (not pulsar) to use an `-M` directive to generate their code with this `Any`.

Reply:
We probably can tell SDK users (with docs) to never import anypb.Any, and use cosmos_proto.Any. But I also see a risk, where they wouldn't be able to use 3rd-party libraries which import anypb.Any. The `-M` directive also seems like additional docs.

So what I realize is that even with this solution, there'll be a lot of documentation. In this case, if we'll end up writing docs, approach 1 in https://github.com/cosmos/cosmos-sdk/issues/13669 seems more desirable, since it's a less invasive solution overall.

Reply:
#92 is another PR that only adds helper functions